OpenRA.Mods.Common/Traits/Transforms.cs

```diff
@@ -103,7 +103,7 @@ public void DeployTransform(bool queued)
 				// Only play the "Cannot deploy here" audio
 				// for non-queued orders
 				foreach (var s in info.NoTransformSounds)
-					Game.Sound.PlayToPlayer(self.Owner, s);
+					Game.Sound.PlayToPlayer(SoundType.World, self.Owner, s);
```

Comment:
Yep.  The way these are used in the mods are a giant mess that needs to be rewritten.  For now, i've assigned the type based on how they are used in the default mods (e.g. all the support power stuff plays UI sounds, most other things play world sounds).